assets/xml/objects/object_boss05.xml

```diff
@@ -1,81 +1,110 @@
+        <Limb Name="gBioDekuBabaHeadUpperJawLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_UPPER_JAW" Offset="0x6260" />
+        <Limb Name="gBioDekuBabaHeadBackUpperLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_BACK_UPPER_LEG" Offset="0x626C" />
+        <Limb Name="gBioDekuBabaHeadBackLowerLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_BACK_LOWER_LEG" Offset="0x6278" />
+        <Limb Name="gBioDekuBabaHeadBackFootLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_BACK_FOOT" Offset="0x6284" />
+        <Limb Name="gBioDekuBabaHeadLeftUpperLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_UPPER_LEG" Offset="0x6290" />
+        <Limb Name="gBioDekuBabaHeadLeftLowerLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_LOWER_LEG" Offset="0x629C" />
+        <Limb Name="gBioDekuBabaHeadLeftFootLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_FOOT" Offset="0x62A8" />
+        <Limb Name="gBioDekuBabaHeadLeftLowerEyestalkLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_LOWER_EYESTALK" Offset="0x62B4" />
+        <Limb Name="gBioDekuBabaHeadLeftUpperEyestalkLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_UPPER_EYESTALK" Offset="0x62C0" />
+        <Limb Name="gBioDekuBabaHeadLeftEyeLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEFT_EYE" Offset="0x62CC" />
+        <Limb Name="gBioDekuBabaHeadRightLowerEyestalkLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_LOWER_EYESTALK" Offset="0x62D8" />
+        <Limb Name="gBioDekuBabaHeadRightUpperEyestalkLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_UPPER_EYESTALK" Offset="0x62E4" />
+        <Limb Name="gBioDekuBabaHeadRightEyeLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_EYE" Offset="0x62F0" />
+        <Limb Name="gBioDekuBabaHeadRightUpperLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_UPPER_LEG" Offset="0x62FC" />
+        <Limb Name="gBioDekuBabaHeadRightLowerLegLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_LOWER_LEG" Offset="0x6308" />
+        <Limb Name="gBioDekuBabaHeadRightFootLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_RIGHT_FOOT" Offset="0x6314" />
+        <Limb Name="gBioDekuBabaHeadLeavesLimb" Type="Standard" EnumName="BIO_DEKU_BABA_HEAD_LIMB_LEAVES" Offset="0x6320" />
+
+        <!-- Bio Deku Baba Head Skeleton -->
+        <Skeleton Name="gBioDekuBabaHeadSkel" Type="Flex" LimbType="Standard" LimbNone="BIO_DEKU_BABA_HEAD_LIMB_NONE" LimbMax="BIO_DEKU_BABA_HEAD_LIMB_MAX" EnumName="BioDekuBabaHeadLimb" Offset="0x6378" />
+
+        <!-- Bio Deku Baba Head Animations -->
+        <Animation Name="gBioDekuBabaHeadChompAnim" Offset="0x6484" /> <!-- Original name is "wdb_pakupaku" ("repeatedly opening and closing (one's mouth)") -->
+        <Animation Name="gBioDekuBabaHeadTransformAnim" Offset="0x6E50" /> <!-- Original name is "wdb_startH" -->
```

Comment:
Transform? BioDekuBaba transforms?

Reply:
This is the animation:
![transform](https://github.com/zeldaret/mm/assets/12245827/259f8135-e0c5-4ce1-900c-a2862c1e8890)

When you detach a Bio Baba from the lily pad, the head still has bits of the lily pad attached to it as it falls. When it hits the ground, the lily pad parts disappear and it plays this animation, For reasons I still haven't quite figured out, though, this animation doesn't really look quite right in-game, so the Bio Baba looks more like the limbs are "popping in" rather than unfolding like this animation.

Definitely a weird animation to describe, so if you have a better term for it, I'm all ears.

Reply:
"Detach" maybe?
But yes, indeed it is hard to name

Reply:
I don't think "detach" is quite right. This video shows what happens when a Bio Baba gets detached. It spawns a Bio Baba with params of 3; once it hits the floor, the lily pad limbs shrink. Once they reach 0.0f scale, it spawns a new Bio Baba with params of 4, which grows the head limbs and plays this animation:

https://github.com/zeldaret/mm/assets/12245827/d8af50af-f357-4698-910f-59768957b53b

You can also see in this video the thing I talked about before; you don't really see much of this animation in-game besides the mouth opening. But in conclusion, this isn't the animation that plays when it gets detached. This is the animation that plays when the detached head *touches the ground*.